misc/scripts/install-local.sh

```diff
@@ -372,14 +365,14 @@ function hashcheck() {
 
 fancy_message info "Retrieving packages"
 mkdir -p "$SRCDIR"
-cd "$SRCDIR"
+cd "$SRCDIR" 2> /dev/null || ( fancy_message warn "Couldn't enter ${SRCDIR}" && exit 1)
 
 case "$url" in
 	*.git)
 		# git clone quietly, with no history, and if submodules are there, download with 10 jobs
 		sudo git clone --quiet --depth=1 --jobs=10 "$url"
 		# cd into the directory
-		cd ./*/ 2> /dev/null
+		cd ./*/ 2> /dev/null || ( fancy_message warn "Couldn't enter into the cloned git repository" && exit 1)
```

Comment:
no smasher thingys. It should be `could not`

Reply:
They are called contractions. Refactoring now.

Reply:
Fixed in https://github.com/pacstall/pacstall/pull/276/commits/0691fb5e7635a168f4f677bc139b3ec1a9983765